pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -435,13 +439,37 @@ def create_model(df_sets, df_parameters, default={}):
         units=model.model_units["volume_time"],
         doc="Water from completions pad storage used for fracturing [volume/time]",
     )
-    model.v_F_UnusedTreatedWater = Var(
+    model.v_F_WaterRemoved = Var(
         model.s_R,
         model.s_T,
         within=NonNegativeReals,
         initialize=0,
         units=model.model_units["volume_time"],
-        doc="Water leftover from the treatment process [volume/time]",
+        doc="Water removed from system post desalination [volume/time]",
```

Comment:
Should this say "post treatment" instead of "post desalination"? Since not all treatment sites are desal sites.

Reply:
@melody-shellman confirmed that this does only apply to desalination sites, not clean brine sites. Recommend renaming the variable to `v_F_DesalinationWaterRemoved` (or something similar) for clarity.